compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -482,7 +482,7 @@ parseProjectConfig project = do
     pName <- queryProjectConfigRequired ["name"] project
     pSrc <- queryProjectConfigRequired ["source"] project
     pExposedModules <- queryProjectConfig ["exposed-modules"] project
-    pVersion <- queryProjectConfigRequired ["version"] project
+    pVersion <- Just <$> queryProjectConfigRequired ["version"] project
```

Comment:
Note that this is not actually changing anything, it just makes it a bit more explicit. I thought you might be able to inject `version: null` in a `daml.yaml` but the check in `queryProjectConfigRequired` treats `null` and the absence of a field identical.